web/controllers/notes_controller.ex

```diff
@@ -2,6 +2,13 @@ defmodule Notox.NoteController do
   use Notox.Web, :controller
   alias Notox.Note
 
+  def index(conn, %{"search" => %{"for" => keyword}}) do
```

Comment:
I don't want to use `query` because it seems confusing in the context of Ecto queries. I'm gonna leave it unless you feel strongly about it.